Makefile.manual

```diff
@@ -1,7 +1,7 @@
 # Fortran stdlib Makefile
 
 FC ?= gfortran
-FFLAGS ?= -Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all
+FFLAGS ?= -Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all  -fno-range-check
```

Comment:
If we are to add ` -fno-range-check` by default for stdlib here, then I suggest adding a comment in the makefile that highlights the reason (e.g. ` # needed for gfortran-9 for hash functions`).

Reply:
Done.